reV/offshore/offshore.py

```diff
@@ -153,14 +153,15 @@ def _parse_cf_meta(gen_fpath):
 
         with Outputs(gen_fpath, mode='r') as out:
             meta = out.meta
+
         if 'offshore' not in meta:
             e = ('Offshore module cannot run without "offshore" flag in meta '
                  'data of gen_fpath: {}'.format(gen_fpath))
             logger.error(e)
             raise KeyError(e)
 
-        onshore_mask = (meta['offshore'] == 0)
-        offshore_mask = (meta['offshore'] == 1)
```

Comment:
This is just a habit of mine. I don't like having more than one equal sign on a line. The parenthesis helps me visually. No worries either way though, just thought i'd note why i do this.

Reply:
But it's not pythonic ;-)

Reply:
I'm not sure about that! May i quote the zen of python: beautiful is better than ugly, and readability counts :)

I'll probably keep doing this haha